kaldb/src/main/java/com/slack/kaldb/logstore/search/SearchResultUtils.java

```diff
@@ -71,7 +67,11 @@ public static <T> KaldbSearch.SearchResult toSearchResultProto(SearchResult<T> s
     span.tag("totalSnapshots", String.valueOf(searchResult.totalSnapshots));
     span.tag("snapshotsWithReplicas", String.valueOf(searchResult.snapshotsWithReplicas));
     span.tag("hits", String.valueOf(searchResult.hits.size()));
-    span.tag("buckets", String.valueOf(searchResult.buckets.size()));
+    span.tag(
+        "internalAggregation",
+        searchResult.internalAggregation != null
+            ? searchResult.internalAggregation.toString()
```

Comment:
can this be massive? if I request bucketCount as 1000 and have some terms aggregations the payload could get big?

Reply:
Yeah, this can be relatively massive. I went ahead and removed it.